src/cli-validator.js

```diff
@@ -12,18 +12,30 @@ function validateArgv(argv) {
   }
 }
 
-function validateIntOption(value, option) {
-  if (!/^\d+$/.test(value)) {
+function getOptionName(type, option) {
+  return type === "cli" ? `--${option.name}` : kebabToCamel(option.name);
+}
+
+function kebabToCamel(str) {
+  return str.replace(/-([a-z])/g, (_, char) => char.toUpperCase());
+}
```

Comment:
This is the third copy we have of this function now. `dashify`, on the other hands, comes from npm. Surely Sindre Sorhus must have a kebabToCamel dependency we can use?

Reply:
https://github.com/sindresorhus/camelcase